src/main/java/generic/Demo.java

```diff
@@ -0,0 +1,31 @@
+package generic;
+
+import circle.Cylinder;
+import shapes.Circle;
+import circle.Ring;
+import shapes.Shape;
+
+import java.util.Arrays;
+
+public class Demo {
+    public static void main(String[] args) {
+        Pair<String, Shape> objectsPair = new Pair<>("Simple Shape", new Shape("red", true));
+        String name = objectsPair.getFirst();
+        Shape shape = objectsPair.getSecond();
+        System.out.println(name);
```

Comment:
`Pair` can have a `toString` implementation that can have both `first` and `second`, so it may be just `System.out.println(objectsPair)` to see what's inside

Reply:
Thanks, will fix it!